backend/corpora/dataset_processing/remaster_cxg.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python3
+import logging
+import subprocess
+
+from backend.corpora.dataset_processing.process import (
+    get_bucket_prefix,
+)
+
+import time
+
+import numpy as np
+import psutil
+import tiledb
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def process(dataset_id: str, cellxgene_bucket: str):
+    """
+    1. Download the labeled dataset from the artifact bucket
+    2. Convert the labeled dataset to CXG
+    3. Upload the CXG to the cellxgene bucket
+    :param dataset_id:
+    :param artifact_bucket:
+    :param labeled_h5ad_filename:
+    :param cellxgene_bucket:
+    :return:
+    """
+
+    # Careful with old datasets - the ones that do not have a UUID
+    # cxg_filename = f"{dataset_id}.cxg"
+
+    # Download the cxg from the bucket
+    bucket_prefix = get_bucket_prefix(dataset_id)
+    object_key = f"{bucket_prefix}.cxg"
+    path = f"s3://{cellxgene_bucket}/{object_key}/X"
+    local_path = "/cxg"
+    logger.info(f"Downloading {path} to {local_path}/X_old")
+
+    download_command = ["aws", "s3", "sync", path, f"{local_path}/X_old"]
+    # Let errors fail the pipeline
+    subprocess.run(download_command, check=True)
+
+    kind = "auto"
+    obs_extent = 256
+    var_extent = 2048
+    cell_order = "row"
+    tile_order = "col"
+    capacity = 128000
+    compression = 22
+    target_array = "X_new"
+    source_array = "X_old"
+    sparse_threshold = 25.0
+
+    compute(
```

Comment:
stylistic nit - do not need to fix if you don't want to do so: I dislike long lists of positional args as it is easy to screw them up in a weakly typed language like Python.  Much prefer kwargs or passing a dict.